package.json

```diff
@@ -26,7 +26,8 @@
         "lint:fix": "eslint --fix src/ --ext=.ts,.tsx",
         "typedoc": "typedoc --mode file --out doc/ --theme minimal --tsconfig tsconfig.json --excludeExternals --externalPattern '**/node_modules/**' --includeDeclarations src/",
         "prettier": "prettier --write \"src/**/*.{ts,tsx}\"",
-        "prettier:check": "prettier --check \"src/**/*.{ts,tsx}\""
+        "prettier:check": "prettier --check \"src/**/*.{ts,tsx}\"",
+        "checks": "npm run lint && npm run prettier:check"
```

Comment:
@dsaxton This PR is for Goban, but something like this what you were talking about doing for the React repo?

Reply:
Indeed, potentially adding TypeScript compilation too if that's seen as useful enough

Reply:
Yeah I think that would be good.  I can do the same for the other repo, and add that TS check (I would find that useful!)